### PR TITLE
Fix example

### PR DESCRIPTION
**Pull request by xepozz**
## Subject
Replaced `permalink` to `url`
Force casted `lastmod` to DATETIME

### Fixed
Example from page https://github.com/sonata-project/SonataSeoBundle/blob/master/docs/reference/sitemap.rst

### CODE REVIEW

File: docs/reference/sitemap.rst [hunk 1]
```diff
@@ -74,8 +74,8 @@ The following code is an extract of the query required to generate a valid sitem
 .. code-block:: sql
 
     SELECT
-        CONCAT_WS('/', YEAR(created_at), MONTH(created_at), DAY(created_at), slug) as permalink ,
-        updated_at as lastmod,
+        CONCAT_WS('/', YEAR(created_at), MONTH(created_at), DAY(created_at), slug) as url ,
+        updated_at as DATETIME(lastmod),
```
Comment (greg0ire) [inline docs/reference/sitemap.rst]: This looks wrong, I don't think you can use functions when creating an alias

File: docs/reference/sitemap.rst [hunk 2]
```diff
@@ -32,8 +32,8 @@ The service must implement the ``SourceIteratorInterface`` from the ``sonata-pro
         {
             $this->key++;
             $this->current = array(
-                'permalink'  => '/the/path/to/target',
-                'lastmod'    => '',
+                'url'  => '/the/path/to/target',
```
Comment (greg0ire) [inline docs/reference/sitemap.rst]: What is wrong with `permalink`?

Comment (xepozz) [reply docs/reference/sitemap.rst]: Please see [one](https://github.com/sonata-project/exporter/blob/2.x/src/Writer/SitemapWriter.php#L222) [two](https://github.com/sonata-project/exporter/blob/2.x/src/Writer/SitemapWriter.php#L265) [three](https://github.com/sonata-project/exporter/blob/2.x/src/Writer/SitemapWriter.php#L282) and other usages in this file. 
 I guess Permalink is outdated name

File: docs/reference/sitemap.rst (hunk 1 above)
Comment (xepozz) [reply docs/reference/sitemap.rst]: You are right.

File: docs/reference/sitemap.rst [hunk 3]
```diff
@@ -108,7 +108,7 @@ Here is a full example coming from the [Sonata Sandbox demo website](https://git
...
                 # blog post
-                - { group: "news",  connection: doctrine.dbal.default_connection, route: sonata_news_view,        parameters: {permalink: null},                        query: "SELECT CONCAT_WS('/', YEAR(created_at), MONTH(created_at), DAY(created_at), slug) as permalink , updated_at as lastmod, 'weekly' as changefreq, '0.5' as priority FROM news__post WHERE enabled = 1 AND (publication_date_start IS NULL OR publication_date_start <= NOW())" }
+                - { group: "news",  connection: doctrine.dbal.default_connection, route: sonata_news_view,        parameters: {url: null},                        query: "SELECT CONCAT_WS('/', YEAR(created_at), MONTH(created_at), DAY(created_at), slug) as url , updated_at as lastmod, 'weekly' as changefreq, '0.5' as priority FROM news__post WHERE enabled = 1 AND (publication_date_start IS NULL OR publication_date_start <= NOW())" }
```
Comment (greg0ire) [inline docs/reference/sitemap.rst]: Let's take this as an occasion to make this readable, shall we ? :sweat_smile: 
```suggestion
                - 
                    group: "news"
                    connection: doctrine.dbal.default_connection
                    route: sonata_news_view
                    parameters: {url: null}
                    query: |
                        SELECT 
                               CONCAT_WS('/', YEAR(created_at), MONTH(created_at), DAY(created_at), slug) as url ,
                               updated_at as lastmod,
                               'weekly' as changefreq,
                               '0.5' as priority
                        FROM news__post
                       WHERE enabled = 1 
                          AND (publication_date_start IS NULL OR publication_date_start <= NOW())" 
```

Comment (xepozz) [reply docs/reference/sitemap.rst]: Yeah, why not. I did expand other objects

**Pull request merged**